product_supplierinfo_state/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+=========================
+Product Supplerinfo State
+=========================
+
+This module provides introduces states to Supplier Pricelists. The default
+states are `Open` and `Closed / Expired`. This state is computed automatically.
+
+By default a Pricelist will be in status `Open`. If a Pricelist end date is
```

Comment:
better a list?

* Default Pricelist Status is `Open`
* When the Pricelist end date is due, the status will change to `Closed / Expired`.